asst1.py

```diff
@@ -0,0 +1,52 @@
+import collections
+
+def swapchars(s):
+  sc = collections.Counter(filter(str.isalpha,s.lower())).most_common()
+  sc2 = list(s)
+  most = sc[0][0]
+  least = sc[len(sc)-1][0]
+  print(sc)
+  count = 0
+  while(count < len(s)):
+    if s[count] == most:
+      sc2[count] = least
+    elif s[count] == most.upper():
```

Comment:
Instead of uppercasing and lowercasing, why not just uppercase or lowercase the entire string at the start?